### DISCUSSION TITLE
txt数据模拟出水深或地形

### DISCUSSION THREAD

**Discussion by cxk-china** [zouyaoji/vue-cesium#370] [Q&A]
![image](https://user-images.githubusercontent.com/73694384/226378142-e3798540-f4c1-4c18-abc0-1e6e6731c244.png)
向大佬请教一下，如何将这种类型的数据使用cesium进行可视化，还请提供一个思路
效果类似下图：
![image](https://user-images.githubusercontent.com/73694384/226378754-12443ee7-9e21-4eec-a60e-2c1c25249f88.png)

**Comment by zouyaoji** [comment]
数据丰富的话可以用热力图组件。
数据不丰富要插值成格点数据来渲染。
